### PR TITLE
Feature/recaptcha check optimization

### PR DESCRIPTION
**Pull request by tuned-up**
Optimization: do not query google if reCaptcha was not submitted

### CODE REVIEW

File: lib/recaptcha/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Recaptcha
-  VERSION = "2.0.0"
+  VERSION = "2.1.0"
```
Comment (grosser) [inline lib/recaptcha/version.rb]: never bump in prs

File: test/verify_test.rb [hunk 2]
```diff
@@ -145,6 +145,25 @@
       assert_nil @controller.flash[:recaptcha_error]
     end
 
+    describe 'when recaptcha response is empty' do
+      before do
+        @controller.params = { 'g-recaptcha-response' => ""}
```
Comment (grosser) [inline test/verify_test.rb]: how does this happen ? `when user did not check the checkbox` ?

File: lib/recaptcha/version.rb (hunk 1 above)
Comment (tuned-up) [inline lib/recaptcha/version.rb]: OK - fixed

File: test/verify_test.rb (hunk 2 above)
Comment (tuned-up) [inline test/verify_test.rb]: Yes, exactly.

**Comment by grosser** [comment]
FYI refactor https://github.com/ambethia/recaptcha/commit/3f53c5c613470548ba318a0b2fb40a24d6848d17

**Comment by grosser** [comment]
thx again, v2.1.0 is out!

**Pull request merged**